Remove [EnabledBySetting=PopoverAttributeEnabled] from ToggleEvent.idl

nt1m opened the pull request:
#### a7a424ab6eb48fee09affb42ac3e1eea7b47b562
<pre>
Remove [EnabledBySetting=PopoverAttributeEnabled] from ToggleEvent.idl
<a href="https://bugs.webkit.org/show_bug.cgi?id=260751">https://bugs.webkit.org/show_bug.cgi?id=260751</a>
rdar://114500635

Reviewed by Alexey Shvayka.

This is now used by details / summary since 267076@main, so the interface should no longer be gated by the popover attribute setting.

* Source/WebCore/dom/ToggleEvent.idl:

Canonical link: <a href="https://commits.webkit.org/267331@main">https://commits.webkit.org/267331@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/b3081bab6866fdd250adce265391c80daba0e2d3

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/16305 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/16624 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/17046 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/18073 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/15292 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/16494 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/19713 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/16742 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/17671 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/16499 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/16938 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/13987 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/18840 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/14183 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/14765 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/21576 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/15170 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/14929 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/18405 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/15519 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/13156 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/14737 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/19103 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/2003 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/15346 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

shvaikalesh reviewed (approved):
Don't we want to emit `ToggleEvent` rather than `Event` in `HTMLDetailsElement::attributeChanged()` as per https://html.spec.whatwg.org/multipage/interactive-elements.html#queue-a-details-toggle-event-task?

nt1m committed:
Remove [EnabledBySetting=PopoverAttributeEnabled] from ToggleEvent.idl
https://bugs.webkit.org/show_bug.cgi?id=260751
rdar://114500635

Reviewed by Alexey Shvayka.

This is now used by details / summary since 267076@main, so the interface should no longer be gated by the popover attribute setting.

* Source/WebCore/dom/ToggleEvent.idl:

Canonical link: https://commits.webkit.org/267331@main
